src/thunderbots/software/radio_communication/main.cpp

```diff
@@ -27,8 +32,32 @@ void primitiveUpdateCallback(const thunderbots_msgs::PrimitiveArray::ConstPtr& m
     {
         primitives.emplace_back(AI::Primitive::createPrimitiveFromROSMessage(prim_msg));
     }
+
+    // Send primitives
+    backend.sendPrimitives(primitives);
 }
 
+void worldUpdateCallback(const thunderbots_msgs::World::ConstPtr& msg)
+{
+    thunderbots_msgs::World world_msg = *msg;
+
+    // Extract team and ball data
+    Team friendly_team = Util::ROSMessages::createTeamFromROSMessage(world_msg.friendly_team);
+    Ball ball = Util::ROSMessages::createBallFromROSMessage(world_msg.ball);
+
+    std::vector<std::tuple<uint8_t, Point, Angle>> detbots;
```

Comment:
`detbots`? Not sure what this means, but I think a more verbose name would help here.

Reply:
renamed